### PR TITLE
Add "Hierarchical routes with attributes" section.

### LINKED ISSUE #175: Add attributes to nested route
I'm not sure if it's the right place to suggest this, so I apologize it is not.
Having playing with  route attributes and hierarchical route and I discover that you It works as excepted, ie , children inherists from parent attributes. However, the syntax is no that straight forward, as the colon needs to be after the attributes. e.g,

/admin AdminR !admin:
   /1 Admin1R !a
   /2 Admin2R !b

then `routeAttrs (Admin1R)` returns `fromList ("admin", "a").

It's a really cool feature which I think could be addd in the route attributes pages.

### PR DESCRIPTION
**Pull request by maxigit**
Fixes #175.

Please feel free to amend and correct anything which needs it.

### CODE REVIEW

File: book/asciidoc/route-attributes.asciidoc [hunk 1]
```diff
@@ -197,3 +197,39 @@ getAdmin3R = defaultLayout [whamlet|Admin3|]
 main :: IO ()
 main = warp 3000 App
 ----
+
+=== Hierarchical routes with attributes
+Of course, you can mix the two approaches. Children of a hierarchical route will inherits the attributes of their parents, e.g.:
```
Comment (snoyberg) [inline book/asciidoc/route-attributes.asciidoc]: s/will inherits/will inherit

File: book/asciidoc/route-attributes.asciidoc [hunk 2]
```diff
@@ -197,3 +197,39 @@ getAdmin3R = defaultLayout [whamlet|Admin3|]
...
+    /1 Admin1R GET !1
+    /2 Admin2R GET !2
+    /3 Admin3R Get !3
+----
+
++AdminR Admin1R+ has the +admin+ and +1+ attributes.
+
+With this technique, you can use the +admin+ attributes in the +isAuthorized+ function, like in the first example.
+You are also sure that you won't forget any attributes as we did with +Admin3R+.
+In this case, this method has no real benefit over the hierarchical route method. However, it becomes more interesting if all your admins pages are not under the same url structures, e.g:
```
Comment (snoyberg) [inline book/asciidoc/route-attributes.asciidoc]: s/In this case/In the following case?

Comment (snoyberg) [inline book/asciidoc/route-attributes.asciidoc]: s/your admins pages/your admin pages

Comment (maxigit) [inline book/asciidoc/route-attributes.asciidoc]: I mean the previous example, so I think the actual wording is correct.

**Pull request merged**